kratos/includes/kratos_filesystem.h

```diff
@@ -57,8 +57,10 @@ namespace FilesystemExtensions {
 std::string KRATOS_API(KRATOS_CORE) CurrentWorkingDirectory();
 
 std::string KRATOS_API(KRATOS_CORE) JoinPaths(const std::vector<std::string>& rPaths);
+  
+std::vector<std::string> KRATOS_API(KRATOS_CORE) ListDirectory(const std::string& rPath);
```

Comment:
Can you add a short docstring
(I know I didn't for my methods, maybe you can add it there too? Cheers!)

Reply:
I will add it

Reply:
done :)